live-build/misc/ansible-roles/appliance-build.virtualization-common/tasks/main.yml

```diff
@@ -45,3 +45,34 @@
       domain=
       dns1=
       dns2=
+
+- name: Add the CLI user with home directory
+  user:
+    name: cli
+    comment: Delphix CLI User
+    uid: 65434
+    non_unique: yes
+    group: staff
+    home: /export/home/cli
+    create_home: yes
+
+- name: Create directory where authorized keys will be specified for all CLI users
+  become_user: cli
+  file:
+    path: /export/home/cli/.ssh
+    state: directory
+
+- name: Configure authorized-keys file names for passwordless SSH into CLI
+  lineinfile:
+    path: /etc/ssh/sshd_config
+    regexp: "^#?{{ item.key }} "
+    line: "{{ item.key }} {{ item.value }}"
+  with_items:
+    - { key: "AuthorizedKeysFile", value: ".ssh/authorized_keys /export/home/cli/.ssh/authorized_keys.%u" }
```

Comment:
should `/export/home/cli/.ssh/authorized_keys.%u` actually be `/export/home/%u/.ssh/authorized_keys.%u`? or even `.ssh/authorized_keys.%u`? perhaps I misunderstand how this value is being used; if so, a comment would be helpful.

Reply:
We don't really need a unique home directory per user. We have always used the `cli` home as a neutral location for authorized-keys files.